docs/content/latest/manage/backup-restore/snapshot-ysql.md

```diff
@@ -128,6 +128,31 @@ This needs to be done for all tablets of all tables in the database.
 cp -r ~/yugabyte-data/node-1/disk-1/yb-data/tserver/data/rocksdb/table-00004000000030008000000000004003/tablet-b0de9bc6a4cb46d4aaacf4a03bcaf6be.snapshots snapshot/
 ```
 
+The file path structure is:
+
+```
+<yb_data_dir>/node-<node_number>/disk-<disk_number>/yb-data/tserver/data/rocksdb/table-<table_id>/[tablet-<tablet_id>.snapshots]/<snapshot_id>
+```
+
+- `<yb_data_dir>` is the directory where YugabyteDB data is stored. (default=`~/yugabyte-data`)
+- `<node_number>` is used when multiple nodes are running on the same server (for testing, QA, and development). The default value is `1`.
+- `<disk_number>` when running yugabyte on multiple disks with the `--fs_data_dirs` flag. The default value is `1`.
+- `<table_id>` is the UUID of the table. You can get it from the Admin UI.
```

Comment:
Add URL in parentheses for "Admin UI" so the user doesn't have to look it up.

Reply:
added